dags/ods/kktix_ticket_orders/udfs/discord_bot.py

```diff
@@ -2,41 +2,109 @@
 import os
 from collections import defaultdict
 from datetime import datetime
-from typing import Dict, Text
+from typing import Dict, List, Text
 
 import requests
 import tenacity
 from airflow.hooks.http_hook import HttpHook
 from airflow.models import Variable
 from google.cloud import bigquery
+from ods.kktix_ticket_orders.udfs.bigquery_loader import TABLE
+from ods.kktix_ticket_orders.udfs.kktix_api import (
+    _get_attendance_book_id,
+    _get_attendee_ids,
+)
 
 DISCORD_HOOK = HttpHook(http_conn_id="discord_registration", method="POST")
+HTTP_HOOK = HttpHook(http_conn_id="kktix_api", method="GET")
 RETRY_ARGS = dict(
-    wait=tenacity.wait_none(),
-    stop=tenacity.stop_after_attempt(3),
+    wait=tenacity.wait_random(min=1, max=10),
+    stop=tenacity.stop_after_attempt(10),
     retry=tenacity.retry_if_exception_type(requests.exceptions.ConnectionError),
 )
+CLIENT = bigquery.Client(project=os.getenv("BIGQUERY_PROJECT"))
 
 
 def send() -> None:
+    refunded_attendee_ids = _check_refunded_ticket_exists()
+    if refunded_attendee_ids:
+        _mark_tickets_as_refunded(refunded_attendee_ids)
```

Comment:
main logic is as follow:
1. _check_refunded_ticket_exists
2. if refunded_attendee_ids, _mark_tickets_as_refunded